backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/businessentities/storage/ImageTransferPhase.java

```diff
@@ -62,4 +62,8 @@ public boolean isPaused() {
     public boolean canBeCancelled() {
         return canBePaused() || isPaused();
     }
+
+    public boolean isFinished() {
+        return this == FINISHED_FAILURE || this == FINISHED_SUCCESS;
+    }
```

Comment:
Nice

Reply:
There are 2 methods in `ImageTransfer`:
1) `isPaused()` - same as `ImageTransferPhase.isPaused()`
2) `isPausedOrFinished()` - the mix of the above `isPaused()` and **your new method**.
IMHO worth (in another PR) to make some code refactoring and remove the 2 from `ImageTransfer`.